app/frontend/components/pages/voting/votingCard.tsx

```diff
@@ -40,7 +40,7 @@ const VotingCard = (): h.JSX.Element => {
   const activeAccount = useActiveAccount()
   const hasRegisteredStakingKey = hasStakingKey(activeAccount)
 
-  const getValidationMessage = (): string => {
+  const getUnmetPreconditionMessage = (): string => {
```

Comment:
I guess this should be
```javascript
  const getUnmetPreconditionMessage = (): string | null => {
```

And I'm wondering - why didn't typescript catch it? :thinking:

Reply:
https://www.amitmerchant.com/strictly-check-null-undefined-typescript/ we have typescript's strictNullChecks off

Reply:
ahhh I see - do you know what's the reason? I'd guess it's meant as a temporary solution as we migrate(d) from TS, so perhaps worth adding an issue for reenabling it in the backlog and fixing whatever violations remain in the code

Reply:
Yes, it affects all the exhaustive ifs and switches as well. It brings a lot of value, but it would be an enormous PR as well.

Reply:
I see, still IMHO probably worth adding explicitly in the backlog to do it eventually, if there's no other workstream as part of the broader type coverage chore

Reply:
Yes of course, I agree, just saying